<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Radar_Info_Widget</class>
 <widget class="QWidget" name="HNS_Radar_Info_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LBL_Radar_Mode_Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Radar Mode:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Radar_Mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>POLICE</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Radar_Info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>25/35/45</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Strobe_Title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Strobes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Strobe_Info">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="text">
    <string>25/5/3</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
